src/app/pages/office/components/vehicle-checks/vehicle-checks-office-card.html

```diff
@@ -0,0 +1,24 @@
+<ion-row *ngIf="display" class="mes-validated-row mes-row-separator ion-nowrap" id="vehicle-checks-card">
+    <ion-col class="component-label ion-align-self-center" size="32">
+        <label>Vehicle Checks</label>
+    </ion-col>
+    <ion-col size="63" class="ion-align-self-center ion-padding-start">
+        <ion-row class="spacing-row"></ion-row>
+        <ion-row *ngFor="let check of checks" class="check-row">
+            <ion-col size="6" class="ion-align-self-start">
+                <div class="counter-icon">
+                    <img *ngIf="!questionHasFault(check)" src="assets/imgs/greenCorrectAnswer.png" alt="Correct Answer"
+                        class="correct-icon icon-image" />
+                    <img *ngIf="questionHasFault(check)" src="assets/imgs/redWrongAnswer.png" alt="Wrong Answer"
+                        class="wrong-icon icon-image" />
+                </div>
+            </ion-col>
+            <ion-col size="3" class="ion-no-padding"></ion-col>
+            <ion-col align-self-start>
```

Comment:
```class="ion-align-self-start"```

Reply:
have updated. cant add screen shot as i dont believe this is part of cat b.